.vale/styles/plain-language-guidelines/use-simple-words-and-phrases.yml

```diff
@@ -0,0 +1,246 @@
+# Vale-code for the list on https://www.plainlanguage.gov/guidelines/words/use-simple-words-phrases/
```

Comment:
Is there any process for how this file can get updated in future? It needs to be copy pasted in like this or can it be pulled dynamically?

Reply:
In that case could this be a GitHub action so that we can write it once and then deploy it in multiple places?

Reply:
good idea

Reply:
I think the lists will probably stay the same. The "Plain language guidelines" project looks stable to me.

In any case, we first need to decide where we want the Vale code to live. Should it live in:

- our main repository `renovatebot/renovate`
- a new repository under our `renovatebot` org
- a repository I control like `honkinggoose/vale-plain-language-guidelines`

Reply:
I think `renovatebot/vale-linter` or something like that

Reply:
How about `renovatebot/vale-plain-language-guidelines`, that should explain that:

- it's for the `vale` program, and 
- it covers the Plain language guidelines

Reply:
Sounds good